### PR TITLE
Remove version from fully qualified name of code generated entities

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/31177
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/31173
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/31991

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-unit-test/src/test/resources/test-src/strand/strand-meta-data.bal [hunk 1]
```diff
@@ -193,3 +194,7 @@ public function assertEquality(any|error expected, any|error actual) {
 public function sleep(int millis) = @java:Method {
     'class: "org.ballerinalang.test.utils.interop.Utils"
 } external;
+
+public function println(any|error... values) = @java:Method {
+    'class: "org.ballerinalang.test.utils.interop.Utils"
+} external;
```
Comment (gimantha) [inline tests/jballerina-unit-test/src/test/resources/test-src/strand/strand-meta-data.bal:200]: newline

File: tests/jballerina-integration-test/src/test/resources/packaging/versions/myPackage/modules/mod1/mod1.bal [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import waruna/http.'client as httpClient;
```
Comment (HindujaB) [inline tests/jballerina-integration-test/src/test/resources/packaging/versions/myPackage/modules/mod1/mod1.bal:1]: Shall we add the license header?

File: tests/jballerina-integration-test/src/test/resources/packaging/versions/myPackage2/main.bal [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import ballerina/test;
```
Comment (HindujaB) [inline tests/jballerina-integration-test/src/test/resources/packaging/versions/myPackage2/main.bal:1]: Shall we add the license header?

File: tests/jballerina-integration-test/src/test/resources/packaging/versions/myPackage2/modules/mod1/mod1.bal [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import waruna/http.'client as httpClient;
```
Comment (HindujaB) [inline tests/jballerina-integration-test/src/test/resources/packaging/versions/myPackage2/modules/mod1/mod1.bal:1]: Shall we add the license header? There are some other `.bal` files as well.

**Review by HindujaB — APPROVED** [review]
LGTM

**Pull request merged**